.gitpod.Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM gitpod/workspace-mysql
+
+USER root
+
+# Update APT Database
+### base ###
+RUN sudo apt-get update -q \
+    && sudo apt-get install -y php-dev
+
+# Install XDebug
+RUN wget http://xdebug.org/files/xdebug-2.9.1.tgz \
+    && tar -xvzf xdebug-2.9.1.tgz \
+    && cd xdebug-2.9.1 \
+    && phpize \
+    && ./configure \
+    && make \
+    && sudo mkdir -p /usr/lib/php/20190902 \
```

Comment:
This line and the next one can be replaced with a simple `make install`

Reply:
thanks for your feedback